packages/browsers/src/httpUtil.ts

```diff
@@ -27,6 +27,7 @@ export function headHttpRequest(url: URL): Promise<boolean> {
       url,
       'HEAD',
       response => {
+        request.destroy();
```

Comment:
We should use `response.resume();` as the issue is that the response is never released.
```suggestion
            // consume response data free node process
            response.resume();
```